packager/packager.cc

```diff
@@ -690,14 +694,41 @@ FullyQualifiedName getFullyQualifiedName(core::Context ctx, const ast::Unresolve
     return fqn;
 }
 
-// Gets the package name in `tree` if applicable.
-PackageName getPackageName(core::Context ctx, const ast::UnresolvedConstantLit *constantLit) {
+PackageName getPackageName(core::Context ctx, const ast::UnresolvedConstantLit *constantLit,
+                           core::ClassOrModuleRef symbol) {
     ENFORCE(constantLit != nullptr);
 
     PackageName pName;
     pName.fullName = getFullyQualifiedName(ctx, constantLit);
 
     // pname.mangledName will be populated later, when we have a mutable GlobalState
+    pName.mangledName.owner = symbol;
```

Comment:
Is this comment still relevant?

Reply:
It's not, and the purpose of this stack of changes is to be able to delete that comment, which I do in the last PR in the stack:

https://github.com/sorbet/sorbet/pull/8851/files#diff-f6e5f987988040a1154edb22bb6607c88d1fc630230056d4af3a4815b58693a1L704

Reply:
(as of this PR, it is still technically relevant)